Frontend/ui-driver/src/Screens/DriverProfileScreen/View.purs

```diff
@@ -268,7 +268,7 @@ profileView push state =
                 , infoView state push
                 ]
             , if state.props.screenType == ST.DRIVER_DETAILS then driverDetailsView push state else vehicleDetailsView push state  -- TODO: Once APIs are deployed this code can be uncommented
-            , if state.props.screenType == ST.DRIVER_DETAILS then payment push state else dummyTextView
+            , if state.props.screenType == ST.DRIVER_DETAILS && (getValueFromConfig "showPaymentDetails") then payment push state else dummyTextView
```

Comment:
Don't use `getValueFromConfig` . Store it state and use.
Define type and add in CommonConfig.purs . Define type for it and use.